pkg/fw/Dockerfile

```diff
@@ -172,6 +178,32 @@ COPY --from=build /rtw88/*.bin /lib/firmware/rtw88/
 COPY --from=build /lib/firmware/intel/ice /lib/firmware/intel/ice
 COPY --from=build /lib/firmware/hailo /lib/firmware/hailo
 
+ARG TARGETARCH
+
+# Remove unnecessary firmware based on TARGETARCH and print freed space
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+    initial_size=$(du -sm /lib/firmware | cut -f1); \
+    rm -rf /lib/firmware/nvidia \
+           /lib/firmware/ti-connectivity \
+           /lib/broadcom \
+           /lib/firmware/*t234* \
+           /lib/firmware/*t194* \
+           /lib/firmware/nvpva* ; \
+    final_size=$(du -sm /lib/firmware | cut -f1); \
+    freed_space=$((initial_size - final_size)); \
+    echo "Freed space: ${freed_space} MB"; \
+    fi
+
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
+    initial_size=$(du -s /lib/firmware | cut -f1); \
+    rm -rf /lib/firmware/intel \
```

Comment:
@OhmSpectator , sorry, I did more checks, we can potentially use firmware from intel folder, it contains firmware for bluetooth devices that we have on arm64 Lenovo SE70, but since we don't support bluetooth, we can keep this removal  if it's strictly needed, otherwise I would like to avoid this removal as well...

Reply:
Ok, sure, I'll keep it.

Reply:
Fixed